src/Functional.CQS.AOP.Caching.Infrastructure.DistributedCache.Redis.Tests/FunctionalRedisCacheTests.cs

```diff
@@ -485,6 +485,8 @@ public override bool Equals(object b)
 
 		#region Data
 
+		private const int CONCURRENT_REQUEST_COUNT = 5;
```

Comment:
Used to be 25 concurrent requests...  The PR build was failing due to Redis timeouts, so I'll have to do some load testing / performance tuning.